packages/server/package.json

```diff
@@ -7,18 +7,18 @@
   "license": "Apache-2.0",
   "scripts": {
     "start": "node src",
-    "start:dev": "npx nodemon src",
-    "serve": "npx nodemon src",
-    "db:migrate": "npx knex migrate:latest",
-    "db:rollback": "npx knex migrate:rollback",
-    "migrate:make": "npx knex migrate:make",
+    "start:dev": "nodemon src",
+    "serve": "nodemon src",
+    "db:migrate": "knex migrate:latest",
+    "db:rollback": "knex migrate:rollback",
+    "migrate:make": "knex migrate:make",
     "env": "export $(cat .env)",
-    "db:seed": "yarn db:migrate && npx knex seed:run",
+    "db:seed": "yarn db:migrate && knex seed:run",
     "test": "yarn test:db && yarn test:email && yarn test:apis",
-    "test:email": "NODE_ENV=test npx mocha __tests__/email/*.test.js",
-    "test:db": "NODE_ENV=test npx mocha --timeout 10000 __tests__/db/*.test.js",
-    "test:apis": "NODE_ENV=test SUPPRESS_EMAIL=TRUE node src > test.log & NODE_ENV=test npx mocha --require __tests__/api/fixtures.js __tests__/api/*.test.js && pkill -fn 'node src'",
-    "lint": "npx eslint '**/*.js'",
+    "test:email": "NODE_ENV=test mocha __tests__/email/*.test.js",
+    "test:db": "NODE_ENV=test mocha --timeout 10000 __tests__/db/*.test.js",
+    "test:apis": "NODE_ENV=test SUPPRESS_EMAIL=TRUE node src > test.log & NODE_ENV=test mocha --exit --timeout 10000 --require __tests__/api/fixtures.js __tests__/api/*.test.js && pkill -fn 'node src'",
```

Comment:
lots of trial and error but seems this + an empty GlobalTeardown is the right combination for CI to work